README.md

```diff
@@ -0,0 +1,33 @@
+# Examining Batch Effects on Cell Painting Data
+
+This project examines the presence of batch effects in (high-throughput) single-cell morphology data of different strains of cells subjected to various perturbations.
+In addition to evaluating the benefits of the kBET detection algorithm, Harmony is used to correct for the observed batch effect. Ultimately, the goal is to benchmark these two processes and evaluate their effectiveness on the LINCS cell morphology data.
```

Comment:
```suggestion
There are two parts to evaluating technical artifacts: Detection and correction.
In this project, we benchmark the kBET detection algorithm, and then use Harmony to correct for the technical artifacts.
Ultimately, the goal is to benchmark these two processes and evaluate their effectiveness in discovering and adjusting for technical artifacts in the [Cancer Cell Morphology Resistance Mechanisms](https://github.com/broadinstitute/profiling-resistance-mechanisms) project.
```

A couple tweaks. It is also good to keep the README to one-sentence-per-line. (this helps with editing)